radix-engine-profiling/src/rocks_db_metrics/tests/read.rs

```diff
@@ -0,0 +1,266 @@
+use super::super::*;
+use super::common::*;
+use radix_engine_store_interface::{
+    db_key_mapper::*,
+    interface::{CommittableSubstateDatabase, DbPartitionKey, DbSortKey, SubstateDatabase},
+};
+use rand::Rng;
+use std::{io::Write, path::PathBuf};
+
+/// Range start of the measuremnts
+const MIN_SIZE: usize = 1;
+/// Range end of the measuremnts
+const MAX_SIZE: usize = 1024 * 1024;
+/// Range step
+const SIZE_STEP: usize = 16 * 1024;
+/// Each step write and read
+const COUNT: usize = 40;
```

Comment:
Can we split `COUNT` into `PREPARE_NUM_OF_SUBSTATES_PER_SIZE` and `READ_NUM_OF_ROUNDS` (feel free to make it shorter if you see fit)?

Reply:
I've added: `PREPARE_NUM_OF_SUBSTATES_PER_SIZE`, `PREPARE_NUM_OF_SUBSTATES_NOT_FUND` and changed `READ_ROUNDS`.